src/sentry/tasks/store.py

```diff
@@ -579,6 +579,12 @@ def _do_save_event(
             # Delete the event payload from cache since it won't show up in post-processing.
             if cache_key:
                 processing_store.delete_by_key(cache_key)
+                if consumer_type == ConsumerType.Transactions:
```

Comment:
this is tracking at the wrong spot. it should be at line 600 where the transactions logic runs.

Reply:
fixed